jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -134,6 +134,30 @@ private static int classIsInterfaceFlag() {
 		return null;
 	}
 
+	public int findElementFromArray(Object array, byte ch, int offset, int length)
+	{
+		for (int i = offset; i < length; i++)
+		{
+			if(getByteFromArrayByIndex(array, i) == ch)
+			{
+				return i;
+			}
+		}
+		return -1;
+	}
+
+	public int findElementFromArray(Object array, char ch, int offset, int length)
+	{
+		for (int i = offset; i < length; i++)
+		{
+			if(getCharFromArrayByIndex(array, i) == ch)
+			{
+				return i;
+			}
+		}
+		return -1;
+	}
```

Comment:
Would it be possible to have consistent coding style with the rest of the file?

Reply:
For sure. Updated.